src/main/java/com/sopt/push/config/EnvConfig.java

```diff
@@ -1,5 +1,27 @@
 package com.sopt.push.config;
 
+import lombok.Getter;
+
+@Getter
 public class EnvConfig {
-  // TODO: Load environment variables
+
+  private static final String DYNAMODB_TABLE_ENV_VAR = "DYNAMODB_TABLE";
+  private static final String ALL_TOPIC_ARN_ENV_VAR = "ALL_TOPIC_ARN";
+
+  private final String dynamoDbTableName;
+  private final String allTopicArn;
+
+  public EnvConfig() {
+    this.dynamoDbTableName = System.getenv(DYNAMODB_TABLE_ENV_VAR);
+    if (this.dynamoDbTableName == null || this.dynamoDbTableName.isBlank()) {
+      throw new IllegalStateException(
+          "Required environment variable '" + DYNAMODB_TABLE_ENV_VAR + "' is not set.");
+    }
+
+    this.allTopicArn = System.getenv(ALL_TOPIC_ARN_ENV_VAR);
+    if (this.allTopicArn == null || this.allTopicArn.isBlank()) {
+      throw new IllegalStateException(
+          "Required environment variable '" + ALL_TOPIC_ARN_ENV_VAR + "' is not set.");
+    }
+  }
```

Comment:
검증로직이 중복되는 것 같아서 검증 로직 메서드 따로 추추할 수 있을 것 같습니당!
생성자는 값만 받구, 검증하는 부분 자체는 따로 로직을 빼는 것도 좋을 것 같다는 생각이 들었어요!

```suggestion
  public EnvConfig() {
      this.dynamoDbTableName = getRequiredEnv(DYNAMODB_TABLE_ENV_VAR);
      this.allTopicArn = getRequiredEnv(ALL_TOPIC_ARN_ENV_VAR);
  }

  private String getRequiredEnv(String key) {
    String value = System.getenv(key);
    if (value == null || value.isBlank()) {
        throw new IllegalStateException(
            "Required environment variable '" + key + "' is not set.");
    }
    return value;
}

```

Reply:
좋은 것 같습니당 ㅎㅎ 좋은 제안 감사합니다 :)